docs/presentation/Content/SelectedNodeInstances.md

```diff
@@ -1,36 +1,175 @@
+| *Misc.*                                                             |
+| [`relatedInstances`](#attribute-relatedinstances)                   | No        | [`RelatedInstanceSpecification[]`](../Common-Rules/RelatedInstanceSpecification.md) | `[]`    |
+
+### Attribute: `acceptableSchemaName`
+
+>> **Default behaviour:** All schema names accepted
+
+Specifies ECSchema name which the input instances have to match for the specification to be used.
+
+```ts
+[[include:SelectedNodeInstances.AcceptableSchemaName.Ruleset]]
+```
+
+  | Selected input                  | Result                                                                                                          |
+  | ------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+  | `BisCore.SpatialViewDefinition` | ![Example when selecting "SpatialViewDefinition"](./media/selectednodeinstances-with-spatialviewdefinition.png) |
+  | `Generic.GroupModel`            | ![Example when selecting "GroupModel"](./media/content-empty-table.png)                                         |
+
+
+### Attribute: `acceptableClassNames`
+
+>> **Default behaviour:** All class names accepted
+
+Specifies a list of class names which the input instances have to match for the specification to be used.
+
+```ts
+[[include:SelectedNodeInstances.AcceptableClassNames.Ruleset]]
+```
+
+  | Selected input                  | Result                                                                                                          |
+  | ------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+  | `BisCore.SpatialViewDefinition` | ![Example when selecting "SpatialViewDefinition"](./media/selectednodeinstances-with-spatialviewdefinition.png) |
+  | `BisCore.DictionaryModel`       | ![Example when selecting "DictionaryModel"](./media/content-empty-table.png)                                    |
+
+### Attribute: `acceptablePolymorphically`
+
+> **Default value:** `false`
+
+Specifies whether derived classes of `acceptableClassNames` should be included in the content.
+
+```ts
+[[include:SelectedNodeInstances.AcceptablePolymorphically.Ruleset]]
+```
+
+  | Selected input                  | Result                                                                                                        |
+  | ------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+  | `BisCore.DictionaryModel`       | ![Example when selecting "DictionaryModel"](./media/selectednodeinstances-with-acceptablepolymorphically.png) |
+  | `BisCore.SpatialViewDefinition` | ![Example when selecting "SpatialViewDefinition"](./media/content-empty-table.png)                            |
```

Comment:
I think a better example would be to have a ruleset with `acceptableClassNames` pointing to a class that has instances of itself and its subclass (not sure if our test imodels have such a case, though). Then we could pass 2 instance keys for the example - one for the class instance and one for the subclass instance. When using `acceptablePolymorphically: true`, content for both instances would be returned. When using `acceptablePolymorphically: false`, only content for the base class instance would be returned.

Reply:
I tried searching for such case but I did not seem to find one. I did update the example to somewhat match the other content specifications when handling polymorphism.

Reply:
That'll do, thanks